app/components/post_body/post_body.js

```diff
@@ -193,6 +195,11 @@ class PostBody extends PureComponent {
             if (canDelete && !hasBeenDeleted) {
                 actions.push({text: formatMessage({id: 'post_info.del', defaultMessage: 'Delete'}), onPress: onPostDelete});
             }
+
+            actions.push({
+                text: formatMessage({id: 'post_info.mobile.add_reaction', defaultMessage: 'Add Reaction'}),
```

Comment:
this id is wrong it should be something like `mobile.post_info.add_reaction` notice the `mobile.` prefix